conftest.py

```diff
@@ -402,3 +414,52 @@ def pytest_sessionfinish(session, exitstatus):
         junit_modifyreport(
             _JSON_REPORT_FILE(), _XML_REPORT_FILE(), junit_properties=context.scenario.get_junit_properties(),
         )
+
+        export_feature_parity_dashbaord(session)
+
+
+def export_feature_parity_dashbaord(session):
+    data = session.config._json_report.report  # pylint: disable=protected-access
+
+    result = {
+        "runUrl": session.config.option.report_run_url,
+        "runDate": data["created"],
+        "environment": session.config.option.report_environment,
+        "testSource": "systemtests",
+        "language": context.scenario.library.library,
+        "variant": context.scenario.weblog_variant,
+        "testedDependencies": [
+            {"name": name, "version": str(version)} for name, version in context.scenario.components.items()
+        ],
+        "scenario": context.scenario.name,
+        "tests": [convert_test_to_feature_parity_model(test) for test in data["tests"]],
+    }
+
+    with open(f"{context.scenario.host_log_folder}/feature_parity.json", "w", encoding="utf-8") as f:
+        json.dump(result, f, indent=2)
+
+
+def convert_test_to_feature_parity_model(test):
+    result = {
+        "path": test["nodeid"],
+        "lineNumber": test["lineno"],
+        "outcome": test["outcome"],
+        "testDeclaration": None,
+        "details": test["skip_reason"],
+        "features": test["metadata"]["features"],
+    }
+
+    if result["details"] is None:
```

Comment:
Currently, I am using the details field to set configuration for the tests, I mean I used this field to set parameters of the tests

Reply:
I noticed that, we'll need to adapt that, as the intent of this field is to provide informations about a potentiel skip/xfail state.

Reply:
then, where can I upload the specific data associated with a test case?